pywbem/_server.py

```diff
@@ -735,31 +735,34 @@ def get_central_instances(self, profile_path, central_class=None,
         * Central class methodology
         * Scoping class methodology
 
-        Use of the scoping class methodology requires specifying the central
-        class, scoping class and scoping path defined by the profile. If any
-        of them is `None`, this method will attempt only the
-        GetCentralInstances and central class methodologies, but not the
-        scoping class methodology.
-        If using these two methodologies does not result in any central
-        instances, and the scoping class methodology cannot be used,
-        :exc:`~pywbem.ModelError` is raised.
-
-        The scoping path is a directed traversal path from the central
-        instances to the scoping instances. Its first list item is always
-        the association class name of the traversal hop starting at the
-        central instances. For each further traversal hop, the list contains
-        two more items: The class name of the near end of that hop, and
-        the class name of the traversed association.
-        As a result, the class names of the central instances and scoping
-        instances are not part of the list.
-
-        Example for a 1-hop traversal:
+        All three methodologies start from the CIM_RegisteredProfile instance
+        referenced by the `profile_path` parameter.
+
+        Autonomous profiles implement the GetCentralInstances methodology
+        and/or the central class methodology.
+        Component profiles implement any one or more of these methodologies.
+
+        The scoping class methodology requires specifying the central class,
```

Comment:
Maybe this clarifies my statement above.

Reply:
I expanded the description so that it no longer requires reading PRP or PUG, and moved it into the Appendix as a new section "Profile advertisement methodologies".